cmd/eks.go

```diff
@@ -182,6 +193,17 @@ If max-retries is unspecified, this command will use a value that translates to
 					waitSleepBetweenRetriesFlag,
 				},
 			},
+			cli.Command{
+				Name:        "cleanup-security-group",
+				Usage:       "Delete the AWS-managed security group created for the EKS cluster.",
+				Description: "When destroying the EKS cluster, the AWS provider leaves behind the security group created for the EKS cluster. This command makes sure to clean up that resource. It can be called before or after the EKS cluster itself is destroyed.",
```

Comment:
NIT: make sure to mention that this will lookup the related SGs by tag, but can also delete SG IDs directly passed in if there are known SG IDs related to the cluster.

Reply:
That would be the ideal way for this to work! I updated it to say what it actually does today, to avoid further churn on this PR. I think we can improve this later. 6503089